lib/src/visitors.dart

```diff
@@ -24,6 +25,8 @@ abstract class SpecVisitor<T> {
 
   T visitClass(Class spec, [T context]);
 
+  T visitExtension(Extension spec, [T context]);
```

Comment:
@matanlurey - is this a breaking change?

Reply:
I guess it depends on the Dart version you use. But with the stable Dart 2.7, it should work. If you use an older Dart version, which doesn't know extension methods, the code generator should work but generates unknown extension methods. see the failing tests

Reply:
It's more of a question of whether `implements SpecVisitor` is an encouraged, or even supported, pattern. It doesn't look like this class is public to the package so it might be safe to call this non-breaking.

In general it can be breaking to add a new method to a class if others may use `implements` with that class.